src/Core/Rexl.Bind/Operations/BuiltinFunctions.cs

```diff
@@ -378,6 +378,9 @@ private BuiltinFunctions()
         AddOne(TextTrimFunc.TrimStart, new Sig(S.AboutTrimStart, A.Create(S.ArgSource, S.AboutTrimStart_Source)));
         AddOne(TextTrimFunc.TrimEnd, new Sig(S.AboutTrimEnd, A.Create(S.ArgSource, S.AboutTrimEnd_Source)));
 
+        AddOne(TextPadFunc.Left, new Sig(S.AboutPadLeft, A.Create(S.ArgSource, S.AboutPad_Source)));
+        AddOne(TextPadFunc.Right, new Sig(S.AboutPadRight, A.Create(S.ArgSource, S.AboutPad_Source)));
```

Comment:
Need an `A.Create` for the "pad to" parameter.

Reply:
The arg name for the second `A.Create` needs to be the correct name, not use `S.ArgSource`.

Reply:
Adding above to new comment and resolving this one.